src/JSInterop/Microsoft.JSInterop/src/JSRuntime.cs

```diff
@@ -54,99 +53,92 @@ protected JSRuntime()
     /// </summary>
     protected TimeSpan? DefaultAsyncTimeout { get; set; }
 
-    /// <summary>
-    /// Invokes the specified JavaScript function asynchronously.
-    /// <para>
-    /// <see cref="JSRuntime"/> will apply timeouts to this operation based on the value configured in <see cref="DefaultAsyncTimeout"/>. To dispatch a call with a different, or no timeout,
-    /// consider using <see cref="InvokeAsync{TValue}(string, CancellationToken, object[])" />.
-    /// </para>
-    /// </summary>
-    /// <typeparam name="TValue">The JSON-serializable return type.</typeparam>
-    /// <param name="identifier">An identifier for the function to invoke. For example, the value <c>"someScope.someFunction"</c> will invoke the function <c>window.someScope.someFunction</c>.</param>
-    /// <param name="args">JSON-serializable arguments.</param>
-    /// <returns>An instance of <typeparamref name="TValue"/> obtained by JSON-deserializing the return value.</returns>
+    /// <inheritdoc/>
+    public JsonSerializerOptions CloneJsonSerializerOptions() => new(JsonSerializerOptions);
```

Comment:
Similar to my previous comment: why expose a bespoke cloning API when this is already possible with STJ APIs?

Reply:
We can't let the shared instance be mutated.

Reply:
Cf. https://github.com/dotnet/aspnetcore/pull/54956#discussion_r1568827075